xapian-core/tests/termgentest.cc

```diff
@@ -735,7 +762,21 @@ static bool test_termgen1()
 	    } else if (strncmp(o, "some", 4) == 0) {
 		o += 4;
 		termgen.set_stemming_strategy(termgen.STEM_SOME);
-	    } else if (strncmp(o, "prefix=", 7) == 0) {
+	    } else if (strncmp(o, "stop=en", 7) == 0) {
+               o += 7;
+               array<const char *, 3> x = {"the", "a", "an"};
```

Comment:
@arnavkj1995 yep, this is brace elision. What you're actually doing is initialising with `{{"the", …}}`, ie initialising the first member of `x` to `{"the", …}`. (And the first member of `array<T>` is the actual array data.)